ncco_helper.py

```diff
@@ -22,3 +22,8 @@ def send_sms(customer_number, text):
             'to': customer_number,
             'text': text,
         })
+
+    @staticmethod
+    def is_alternatives_in_limit(alternatives, freed_slot, limit=2):
+        for alternative in alternatives:
+            return alternative[0] + limit == freed_slot or alternative[0] - limit == freed_slot
```

Comment:
1) loop will exit on first comparison
2) no need to do +- limit, since it is done when generating alternatives